apps/api/.env

```diff
@@ -5,4 +5,4 @@
 # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
 NODE_ENV=development
-DATABASE_URL=postgresql://dumpovac:dump123@localhost:3001/internship-app
```

Comment:
ove promjene koje ovise o tvojoj lokalnoj konfiguraciji trebaju ici u `.env.local`

Reply:
ovo je postavljeno u `.env` jer su to postavke za bazu kad se digne sa `docker compose up` a za svoje lokalne promjene napravis `.env.local` file u istom folderu koji je u gitignoreu i u njemu izmjenis na `DATABASE_URL` koji tebi triba

Reply:
aha ok ond